### ISSUE TITLE
Back Button

### ISSUE THREAD

**Issue by trevore** [oculusinfo/TellFinder#14]
CEOS users would like the ability to go back to a previous entity search. If one is viewing an entity's Entity Graph and double clicks on a phone number, image, or any other attribute, then a new search is conducted. There is no way to go back to the previous entity search except to re-search the "Enter Tip" field and then select the ad group from the results list.

**Comment by eric-oculus** [comment]
Agreed. Operations like search that pull up entirely new information should cause a redirect with information stored in the URL so that the back button will work.
